### PR TITLE
Group controller

### PR DESCRIPTION
**Pull request by Redvin-dt**

### CODE REVIEW

File: server/src/main/java/ru/hse/server/service/GroupService.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    static private final Logger logger = LoggerFactory.getLogger(GroupService.class);
+    private final GroupRepository groupRepository;
+    private final UserRepository userRepository;
+
+    public GroupService(GroupRepository groupRepository, @Qualifier("userDatabaseRepository") UserRepository userRepository) {
+        this.groupRepository = groupRepository;
+        this.userRepository = userRepository;
+    }
+
+    public void createGroup(GroupModel groupModel) throws InvalidProtocolBufferException {
```
Comment (YuriiFilatov) [inline server/src/main/java/ru/hse/server/service/GroupService.java]: Group must return protobuf with information of new group

**Review by romanpovol — APPROVED** [review]

File: server/src/main/java/ru/hse/server/controller/GroupController.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        }
+    }
+
+    @GetMapping(produces = {MediaType.APPLICATION_PROTOBUF_VALUE, MediaType.TEXT_PLAIN_VALUE})
+    public ResponseEntity getGroup(@RequestParam Long id) {
+        try {
+            var group = groupService.findGroupById(id);
+            return ResponseEntity.ok().body(group);
+        } catch (EntityNotFoundException e) {
+            logger.error("can not get grop with id={}, since that group does not exists", id);
```
Comment (YuriiFilatov) [inline server/src/main/java/ru/hse/server/controller/GroupController.java]: typo in word group

File: server/src/main/java/ru/hse/database/entities/Group.java [hunk 3]
```diff
@@ -12,6 +15,7 @@
 @Table(name = "Groups")
 @Getter
 @Setter
+@JsonIgnoreProperties(value = {"chapters", "usersSet", "admin"}) // TODO: write own serialization/deserialization
```
Comment (YuriiFilatov) [inline server/src/main/java/ru/hse/database/entities/Group.java]: We use protobuf, useless annotation

File: server/src/main/java/ru/hse/database/entities/User.java [hunk 4]
```diff
@@ -11,6 +12,7 @@
 @Table(name = "Users")
 @Getter
 @Setter
+@JsonIgnoreProperties(value = {"groupsAdmin", "groupsUserSet"}) // TODO: write own serialization/deserialization
```
Comment (YuriiFilatov) [inline server/src/main/java/ru/hse/database/entities/User.java]: We use protobuf, useless annotation

**Pull request merged**